apps/admin-portal/package.json

```diff
@@ -36,7 +36,8 @@
         "@wso2is/react-components": "^1.0.313",
         "@wso2is/theme": "^1.0.313",
         "@wso2is/validation": "^1.0.313",
-        "country-language": "^0.1.7"
+        "country-language": "^0.1.7",
+        "redux-form": "^8.3.6"
```

Comment:
Shall we move this to the root `package.json` and add this under the peer dependencies hash here?